nexus_constructor/treeview_utils.py

```diff
@@ -134,11 +159,12 @@ def handle_number_of_items_selected_is_not_one(
     :param create_link_action: The action for creating a link.
     :param zoom_action: The action for zooming on a component.
     """
-    delete_action.setEnabled(False)
-    new_rotation_action.setEnabled(False)
-    new_translation_action.setEnabled(False)
-    create_link_action.setEnabled(False)
-    zoom_action.setEnabled(False)
+    set_enabled_and_raise(delete_action, False)
+    set_enabled_and_raise(new_rotation_action, False)
+    set_enabled_and_raise(new_translation_action, False)
+    set_enabled_and_raise(create_link_action, False)
+    set_enabled_and_raise(zoom_action, False)
+    set_enabled_and_raise(edit_component_action, False)
```

Comment:
Again, 6 lines of same function call. And all of the cases second argument is the same.

Reply:
good point. This is fixed now,